### PR TITLE
Update Integration descriptions

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Reminder to self to rebase merge this.

### CODE REVIEW

File: src/sentry/integrations/jira/integration.py [hunk 1]
```diff
@@ -1,37 +1,59 @@
...
-    Integration, IntegrationFeatures, IntegrationProvider, IntegrationMetadata
+    Integration, IntegrationFeatures, IntegrationMetadata, IntegrationProvider,
+)
+from sentry.integrations.exceptions import (
+    ApiError, ApiUnauthorized, IntegrationError,
 )
-from sentry.integrations.exceptions import ApiUnauthorized, ApiError, IntegrationError
 from sentry.integrations.issues import IssueSyncMixin
 from sentry.utils.http import absolute_uri
+from six.moves.urllib.parse import quote_plus
```
Comment (EvanPurkhiser) [inline src/sentry/integrations/jira/integration.py]: isort seems to be a little wonky, will remove these reorder changes

File: src/sentry/integrations/slack/integration.py [hunk 2]
```diff
@@ -11,28 +11,31 @@
...
 
 DESCRIPTION = """
-Define a relationship between Sentry and your Slack workspace(s).
+Connect your Sentry organization to one or more Slack workspaces, and start
+getting errors right in front of you where all the action happens in your
+office!
 
- * Unfurls Sentry URLs in slack, providing context and actionability on issues
-   directly within your Slack workspace.
+ * Unfurls Sentry URLs in directly within Slack, providing you context and
```
Comment (MeredithAnya) [inline src/sentry/integrations/slack/integration.py]: I think you don't need that 'in'

File: src/sentry/integrations/slack/integration.py [hunk 3]
```diff
@@ -11,28 +11,31 @@
 from sentry.utils.http import absolute_uri
 
 DESCRIPTION = """
-Define a relationship between Sentry and your Slack workspace(s).
+Connect your Sentry organization to one or more Slack workspaces, and start
+getting errors right in front of you where all the action happens in your
+office!
```
Comment (MeredithAnya) [inline src/sentry/integrations/slack/integration.py]: so much action ! 💥

**Review by MeredithAnya — APPROVED** [review]
I think all the words look good.

**Pull request merged**